src/main/java/Duke.java

```diff
@@ -1,10 +1,121 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static String line = "\t________________________________________";
```

Comment:
constant terms should be named in all caps, a different name would be also preferrable